frontend/src/components/Admin/AdminShoutouts/AdminShoutouts.tsx

```diff
@@ -1,41 +1,92 @@
 import React, { useState, useEffect } from 'react';
-import { Item, Card } from 'semantic-ui-react';
+import { Form, Item, Card } from 'semantic-ui-react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import { Emitters } from '../../../utils';
 import { ShoutoutsAPI, Shoutout } from '../../../API/ShoutoutsAPI';
 import styles from './AdminShoutouts.module.css';
 
 const AdminShoutouts: React.FC = () => {
   const [shoutouts, setShoutouts] = useState<Shoutout[]>([]);
+  const [displayShoutouts, setDisplayShoutouts] = useState<Shoutout[]>([]);
   useEffect(() => {
-    ShoutoutsAPI.getAllShoutouts().then((shoutouts) => setShoutouts(shoutouts));
+    ShoutoutsAPI.getAllShoutouts().then((shoutouts) => {
+      setShoutouts(shoutouts);
+      setDisplayShoutouts(shoutouts);
+    });
   }, []);
+  const [earlyDate, setEarlyDate] = useState<Date>(new Date('2022/01/01'));
+  const [lastDate, setLastDate] = useState<Date>(new Date());
 
   const fromString = (shoutout: Shoutout): string => {
+    const dateString = ` (Date: ${new Date(shoutout.timestamp).toDateString()})`;
     if (!shoutout.isAnon) {
       const { giver } = shoutout;
-      return `From: ${giver?.firstName} ${giver?.lastName} (${giver.email})`;
```

Comment:
Just want to double-check that we agreed on deleting the email/NetID from the text as part of this change too, since I didn't see a note about that in the PR or the notion page. Feel free to ignore this if you did agree about that.

Reply:
This is connected to not making the shoutouts page too long for admins to scroll through - I exchanged the giver's email for the date, but this can be undone if we want the email. @JacksonStaniec do you know if leads definitely want the giver's email?

Reply:
Makes sense. I totally agree with getting rid of it by the way. I just wanted to make sure we formally agreed on it.

Reply:
We're moving to make the giver's actual identity only relevant on the backend, for content moderation purposes. So, their netid doesn't need to be displayed. Just so we're all on the same page, I've asked @airaagrawal1 to feel free to take more creative liberty with the Shoutouts feature because I think she's got good ideas for it, so it's alright if there's deviations from the original ticket. @andrew032011 I do think its a good idea to document these changes somewhere if they are major

Reply:
Ok, I’m on board with the decision and will keep that in mind for the rest of the review. Thanks for letting me know.

Reply:
Sounds good, will make sure to check in next time before making a big change.